packages/oscal-react-library/src/components/images/icons/delete.svg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/

```

Comment:
For all custom `svg` files - we should just be grabbing the `svg`s directly (with their path code) and placing them into this images folder rather than importing this through a link on figma.

Reply:
Done.